### DISCUSSION TITLE
Connecting to conserver via a telnet client

### DISCUSSION THREAD

**Discussion by juergh** [bstansell/conserver#51] [Q&A]
I can successfully connect to a conserver console using a telnet client and issue commands but I don't seem to be getting any responses back from the OS. I also have a 'console' instance connected in spy mode which shows the output of the command that was issued via the telnet client. For example, I send 'ls' via the telnet client and the output of that command only shows up in the 'console' instance but not the telnet client. What am I missing?

**Comment by juergh** [comment]
It seems that I need to send '^Ec;' to receive responses. Why is that?

**Comment by bstansell** [comment]
Sorry for not getting back sooner, but, you're right - looks like that sequence is needed.  I haven't gone back through the code history enough to remind me _why_ this is the case, but the client does send that code once a "call" has been made to the console.  It all integrates into the concept of a client actively "connected" or not to the console (there needed to be a way to suspend output to clients based on what's happening on the client) and, for some reason, clients are put into the unconnected state upon the "call".  The `PROTOCOL` file certainly doesn't document this and either needs updating or the initial state reversed.  I did a quick test and things totally broke, so there is some other piece of code that is assuming things as well.  It's been this way for so long, I think the `PROTOCOL` doc update is the path I'll go down.
